### PR TITLE
Improve install docs

### PR DESCRIPTION
**Pull request by mikeavena**
| Q                  | A
| ------------- | ---
| Doc fix?         | yes
| New docs?    | yes, installing-publisher.rst
| Applies to     | 2.0
| Fixed tickets  | 

Changes include adding a new installation chapter, updated documentation for connecting Superdesk and Superdesk Publisher, and updated configuration docs for Publisher 2.0 and security features. All documentation was tested for accuracy using Superdesk Publisher 2.0.3.

License: AGPLv3

### CODE REVIEW

**Comment by takeit** [comment]
Hi, @mikeavena thank you! We would need you to sign the CLA. Could you do that please? https://www.sourcefabric.org/en/home/policies/3201/Contributor-License-Agreement.htm

**Comment by mikeavena** [comment]
Hi, @takeit I just signed and sent the agreement over.

**Comment by bfkolle** [comment]
@mikeavena it might be beneficial to add the endpoint to get the api documentation 'http://<domain_name>/api/doc/' somewhere

**Comment by ahilles107** [comment]
Thanks a lot for this contribution! It is awesome. And we will definetly aprove it and merge. 
Before that let's fix issues found by sphinx on docs build (connected to changes):
```
/Users/pawelmikolajczuk/Projects/web-publisher/docs/manual/getting_started/installing-publisher.rst:110: WARNING: Error in "code-block" directive:
1 argument(s) required, 0 supplied.

.. code-block::

        server {
                server_name example.com
                listen 80 default;
                root /var/www/publisher/public;

                location / {
                        try_files $uri /index.php$is_args$args;
                }

                location ~ ^/index\.php(/|$) {
                        fastcgi_pass 127.0.0.1:9000;
                        fastcgi_split_path_info ^(.+\.php)(/.*)$;
                        include fastcgi_params;

                        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
                        fastcgi_param DOCUMENT_ROOT $realpath_root;

                        internal;
                }

                location ~ \.php$ {
                        return 404;
                }

                error_log /var/log/nginx/project_error.log;
                access_log /var/log/nginx/project_access.log;
        }
/Users/pawelmikolajczuk/Projects/web-publisher/docs/manual/getting_started/publisher-configuration.rst:15: WARNING: Error in "code-block" directive:
1 argument(s) required, 0 supplied.

.. code-block::

        APP_ENV=prod
        APP_DEBUG=0
/Users/pawelmikolajczuk/Projects/web-publisher/docs/manual/getting_started/publisher-configuration.rst:28: WARNING: Error in "code-block" directive:
1 argument(s) required, 0 supplied.

.. code-block::

        SWP_DOMAIN=example.com
/Users/pawelmikolajczuk/Projects/web-publisher/docs/manual/getting_started/publisher-configuration.rst:39: WARNING: Error in "code-block" directive:
1 argument(s) required, 0 supplied.

.. code-block::

        APP_SECRET=pasteYourSecretKeyHere
/Users/pawelmikolajczuk/Projects/web-publisher/docs/manual/getting_started/publisher-configuration.rst:56: WARNING: Error in "code-block" directive:
1 argument(s) required, 0 supplied.

.. code-block::

        DATABASE_URL=pgsql://root:yourPasswordHere@127.0.0.1/publisher_%kernel.environment%?charset=utf8&serverVersion=9
/Users/pawelmikolajczuk/Projects/web-publisher/docs/manual/getting_started/superdesk-configuration.rst:96: WARNING: Inline emphasis start-string without end-string.

/Users/pawelmikolajczuk/Projects/web-publisher/docs/manual/getting_started/navigation-management.rst:4: WARNING: unknown document: /manual/editorial_tools/livesite_editor
/Users/pawelmikolajczuk/Projects/web-publisher/docs/manual/getting_started/superdesk-superdesk-publisher-setup.rst:86: WARNING: Pygments lexer name u'env' is not known
```

File: docs/manual/getting_started/superdesk-superdesk-publisher-setup.rst [hunk 1]
```diff
@@ -1,403 +1,113 @@
...
+To add this component as a dependency edit ``/opt/superdesk/client/package.json`` and include the 
+Superdesk Publisher repository:
 
-Inside ``/var/www/publisher`` directory, run the command to create the database:
+.. code-block:: js
 
-.. code-block:: bash
+	"dependencies": {
+		....,
+		"superdesk-publisher": "superdesk/superdesk-publisher#master"
```
Comment (ahilles107) [inline docs/manual/getting_started/superdesk-superdesk-publisher-setup.rst]: how about changing it to `superdesk/superdesk-publisher#2.0` ? Master likes to be unstable ;)

File: docs/manual/getting_started/superdesk-superdesk-publisher-setup.rst [hunk 2]
```diff
@@ -1,403 +1,113 @@
...
-.. code-block:: bash
-
-    sudo -u www-data SYMFONY_ENV=prod php bin/console sylius:theme:assets:install
+Login to the server where Publisher is installed.
 
-8. Run supervisor
------------------
-
-.. code-block:: bash
+Edit (or create, if it doesn't already exist) the file ``/var/www/publisher/.env.local`` and add the following:
```
Comment (ahilles107) [inline docs/manual/getting_started/superdesk-superdesk-publisher-setup.rst]: loosk like sphinx didn't get this code block (`/var/www/publisher/.env.local`).

**Comment by mikeavena** [comment]
@ahilles107 I can make these changes as requested. Is there a preferred code-block directive for environment and configuration files? I didn’t want to use bash and I thought leaving it blank would be fine for a generic.

**Comment by ahilles107** [comment]
@mikeavena I don't know any code bloc directive for env files. I think that you can set it to `default`

**Comment by mikeavena** [comment]
I fixed all the issues noted here. Apologies for the delay - I was away for the holiday weekend.

**Comment by ahilles107** [comment]
Thank You for your work!

**Pull request merged**